core/src/main/java/org/apache/calcite/util/RelToSqlConverterUtil.java

```diff
@@ -64,6 +68,79 @@ public static void unparseHiveTrim(
     }
   }
 
+  /**
+   * Unparses IS_TURE/IS_FALSE/IS_NOT_TRUE/IS_NOT_FALSE to right format for dialects.
+   *
+   * <p>For example :
+   *
+   * <blockquote><pre>
```

Comment:
1)Unparses IS TRUE,IS FALSE,IS NOT TRUE and IS NOT FALSE.
2) Generalize this expression using A( A IS NULL ), and explain that an exception will be thrown when A is a non-deterministic function.

Reply:
Had changed, please help to review with the comments whether is OK, thanks~ @NobiGo